packages/matchbox/src/components/RadioCard/RadioCard.tsx

```diff
@@ -51,28 +54,35 @@ const RadioCard = React.forwardRef<HTMLInputElement, RadioCardProps>(function Ra
         ref={userRef}
         type="radio"
         value={value}
+        {...rest}
       />
-      <StyledLabel as="label" htmlFor={id}>
+      <StyledLabel as="label" htmlFor={id} $size={size}>
         <Box display="flex">
-          <Box flex="0 0 auto" width="400">
-            <Box position="absolute" top="500" left="500">
+          <Box flex="0 0 auto" width={isSmall ? '200' : '400'}>
+            <Box position="absolute" top={isSmall ? '100' : '500'} left={isSmall ? '200' : '500'}>
               <StyledUnchecked size="1rem" as={RadioButtonUnchecked} />
               <StyledChecked size="1rem" as={RadioButtonChecked} />
             </Box>
           </Box>
           <Box flex="1" pl="300">
             <StyledHeader
               data-id="radio-card-header"
-              fontSize={fontSize}
-              lineHeight="400"
-              fontWeight="semibold"
+              fontSize={isSmall ? '200' : fontSize}
```

Comment:
Maybe we rethink how fontSize is being defined, having `'200' | fontSize` is confusing

Reply:
@logansparlin Do you think I should just remove the `weight` prop? We don't use it anywhere in the app, its not in the figma component, and not entirely sure why we set it up like this